wireapp/wire-android
# feat: Show conversation Proteus verification status

# What's new in this PR?

### Issues

When the conversation is "Proteus verified" user should know about it: corresponding icon next to the conversation name and systemMessage in a conversation + systemMessage when the conversation becomes Degraded (was verified in past but not anymore).

Needs releases with:

- wireapp/kalium/pull/2157

#### How to Test

1. 
- open some users profile
- go to Devices tab
- verify all the devices of the user
- go to the conversation with that user
expected result: 
there should be a system message informing that conversation is Verified now + blue shield icon is next to the user name 
- go to the Conversations List screen 
expected: 
blue shield icon is next to the user name and second line in conversation item is SystemMessage

2. 
- open some group conversation
- verify all the devices of all the users in that group (see the prev. test)
- go to that conversation
expected result: 
there should be a system message informing that conversation is Verified now + blue is next to the conversation name
- go to the Conversations List screen 
expected: 
blue shield icon is next to the conversation name and second line in conversation item is SystemMessage

3. 
- do all from test-2
- un-verify at least 1 device of 1 user
- go to that conversation
expected result: 
there should be a system message informing that conversation is NOT Verified anymore + NO blue is next to the conversation name
- go to the Conversations List screen 
expected: 
NO blue shield is next to the conversation name and second line in conversation item is SystemMessage

### Attachments (Optional)

<img width="286" alt="Screenshot 2023-10-27 at 11 40 09" src="https://github.com/wireapp/wire-android-reloaded/assets/6539347/3d8d35cf-9532-450c-a580-fb4cc9beb132">

<img width="286" alt="Screenshot 2023-10-27 at 11 40 21" src="https://github.com/wireapp/wire-android-reloaded/assets/6539347/3c69c3c5-c5bb-438d-84c2-1d78054271a2">

<img width="285" alt="Screenshot 2023-10-19 at 17 59 17" src="https://github.com/wireapp/wire-android-reloaded/assets/6539347/f025d893-3741-41cd-a398-f06bfab7b318">

<img width="289" alt="Screenshot 2023-10-19 at 17 59 53" src="https://github.com/wireapp/wire-android-reloaded/assets/6539347/7f17e757-c4ee-45d1-a68f-9b6f7281a3dd">

<img width="281" alt="Screenshot 2023-10-19 at 17 59 35" src="https://github.com/wireapp/wire-android-reloaded/assets/6539347/8227f731-543e-4ac9-aa52-6b8724a3976e">
